src/Utils.php

```diff
@@ -190,16 +190,19 @@ public static function naturalLanguageJoinWithBackticks(array $names)
 
     /**
      * Handle triggering deprecation error.
-     *
-     * @param string $message
-     * @param string $exceptionClass
      */
-    public static function triggerDeprecation($message, $exceptionClass = \RuntimeException::class)
+    public static function triggerDeprecation(\Exception $futureException)
     {
         if (getenv('PHP_CS_FIXER_FUTURE_MODE')) {
-            throw new $exceptionClass("{$message} This check was performed as `PHP_CS_FIXER_FUTURE_MODE` env var is set.");
```

Comment:
TLDR - this was not working for `InvalidFixerConfigurationException` which has a constructor prototype of `$name, $description` and not `$description` only